backend/overstroomik_service/main.py

```diff
@@ -17,24 +25,60 @@
 
 @app.get("/")
 async def read_root():
-    return {"Hello": "World"}
+    return Webservice(version=settings.WS_VERSION, status=settings.ERROR_GENERAL_NOER)
+
+
+@app.get('/by_rd', response_model=FloodInfo)
+async def by_rd(x: Optional[float] = None, y: Optional[float] = None) -> FloodInfo:
+
+    # initieel no error
+    status = settings.ERROR_GENERAL_NOER
+
+    # Convert rd to lat/lon
+    rijksdriehoek = Rijksdriehoek()
+    latlon = rijksdriehoek.rd_to_wgs(x, y)
+    print (len(latlon))
+    # Get address information from the PDOK services
+    pdok = PDOK()
+    status, location = await pdok.get_address(searchfield=None, latitude=latlon[0], longitude=latlon[1])
```

Comment:
I think the idea of this backup endpoint is to skip PDOK, because it has been done by the frontend already. So we only need the geoserver part.

Reply:
correct, i changed the code